src/components/Field/Field.tsx

```diff
@@ -70,14 +95,27 @@ export class Field extends React.PureComponent<FieldProps> {
     return (
       <div className={classes}>
         {label ? <Header sub>{label}</Header> : null}
-        <Input
-          value={value}
-          type={isAddress ? 'text' : type}
-          icon={icon}
-          loading={loading && !isAddress}
-          disabled={disabled}
-          {...rest}
-        />
+        {type === DATE_TYPE ? (
+          <SemanticDatepicker
+            value={value && new Date(`${value} 00:00:00`)}
```

Comment:
Would you mind adding a simple comment here saying that we have added the time to the date to prevent timezone variations that would change the date?
```suggestion
            value={value && new Date(`${value} 00:00:00`)}
```